docs/content/guides/dagster/dagster_type_factories.mdx

```diff
@@ -0,0 +1,285 @@
+---
+title: Validating Data with Dagster Type Factories | Dagster
+description: This guide illustrates the use of a Dagster Type factory to validate Pandas dataframes using the third-party library Pandera.
+---
+
+# Validating Data with Dagster Type Factories
```

Comment:
It would be helpful to include a high-level summary of what the guide covers at the top - maybe just copy/paste the description from the page metadata.

Reply:
I already have this sentence: "In this guide we'll illustrate a technique for integrating third-party data validation libraries into your Dagster jobs: the [Dagster Type](/concepts/types) factory pattern."

Do you think I should specifically mention pandera/pandas at the top? I thought I should avoid this, for the reason suggested in your high-level comment-- to avoid making it look like this guide is *about* pandas dataframes/pandera, especially since dagster-pandera will be up shortly.

Reply:
The "Warning" section looks great.